geonode/services/templates/services/service_detail.html

```diff
@@ -95,23 +96,13 @@ <h3>{% trans "Service Resources" %} <span class="badge">{{ total_resources }}</s
 </div>
 {% endblock %}
 {% block sidebar %}
-
-
-    {% get_obj_perms request.user for service as "resource_perms" %}
```

Comment:
This change here seems to be outside the scope of the PR.

Which is the intent of this change?

Reply:
At the beggining i thought services also are included in the advanced pubblish workflow. so i did some preparations in this view too. Will restore it to previous state